ecosystem/platform/server/app/controllers/onboarding_controller.rb

```diff
@@ -52,8 +52,7 @@ def kyc_callback
       redirect_to it1_path, notice: 'Identity Verification completed successfully!'
     rescue KYCCompleteJobError => e
       Sentry.capture_exception(e)
-      redirect_to it1_path, error: 'Error; If you completed Identity Verification,'\
-                                   " it may take some time to reflect. Error: #{e}"
```

Comment:
is there a way we can just put the error message?

```
    raise KYCCompleteJobError, "Could not get inquiry '#{inquiry_id}' for user ##{user.id}" unless inquiry.present?
    raise KYCCompleteJobError, "Inquiry '#{inquiry_id}' reference_id did not match expected user ##{user.id}"
    raise KYCCompleteJobError, "Inquiry was not complete! Status: '#{status}'" unless status == 'completed'
```

None of the fields that get filled in should ever be large

Is it possible that any param args end up in the cookie data?